spec/lutaml/parsers/xmi_251_spec.rb

```diff
@@ -262,17 +253,17 @@
               {
                 definition: nil,
                 name: "requirement",
-                type: "extension_Bibliography",
+                type: nil,
               },
               {
                 definition: nil,
                 name: "recommendation",
-                type: "extension_Bibliography",
+                type: nil,
               },
               {
                 definition: nil,
                 name: "premission",
```

Comment:
This is "permission"?

Reply:
This value is from line 61 of the file`spec/fixtures/ea-xmi-2.5.1.xmi`.

```xml
<ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="EAID_547E848A_0513_4565_8B45_495BC23F9B3D" name="premission"/>
```

Reply:
Got it. Can you change both to the correct spelling? Thanks.

Reply:
OK.  I will correct the spelling.